docs/api/app.md

```diff
@@ -904,6 +904,15 @@ Returns [`ProcessMetric[]`](structures/process-metric.md): Array of `ProcessMetr
 
 Returns [`GPUFeatureStatus`](structures/gpu-feature-status.md) - The Graphics Feature Status from `chrome://gpu/`.
 
+### `app.getGPUInfo([callback])`
+
+Returns `Object` - GPU Information available in chromium's GPUInfo object. It's contains the version and driver information that's shown on `chrome://gpu` page.
+
+* `callback` Function (optional)
+  * `gpuInfo` Object - List of all attributes and values available in [chromium's GPUInfo object](https://chromium.googlesource.com/chromium/src/gpu/+/master/config/gpu_info.cc).
```

Comment:
Can you please use a link to a Chromium release tag, not to a `master`?
That file might get renamed or removed.

Reply:
Yes, I thought of doing that. We don't do that in other places in the docs though. Also, we should add another item on the release checklist to update the docs.